static/app/views/organizationGroupDetails/grouping/grouping.tsx

```diff
@@ -80,6 +80,8 @@ function Grouping({api, groupId, location, organization, router, projSlug}: Prop
 
   useEffect(() => {
     fetchGroupingLevels();
+    const unListener = browserHistory.listen(handleRouteLeave);
+    return () => unListener();
```

Comment:
This can be simplified to 
```suggestion
    return browserHistory.listen(handleRouteLeave);
```